lib/streamlit/app_session.py

```diff
@@ -105,15 +106,9 @@ def __init__(
         # due to the source code changing we need to pass in the previous client state.
         self._client_state = ClientState()
 
-        # The script should rerun when the `secrets.toml` file has been changed.
-        secrets._file_change_listener.connect(self._on_secrets_file_changed)
+        self._local_sources_watcher: LocalSourcesWatcher = local_sources_watcher
```

Comment:
Micronit: I think this type annotation is unnecessary because `local_sources_watcher` (the function parameter) is already typed.